examples/compareKernelTypes.py

```diff
@@ -49,70 +51,70 @@
 defSciencePath = None
 defTemplatePath = None
 
+
 class DiffimTestCases(unittest.TestCase):
```

Comment:
Should I move them to the tests directory or just leave them in examples? I don't have a good feel for how many of each are typical.